src/hr/hr.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Controller, Post, Body, HttpStatus, HttpCode } from '@nestjs/common';
+import { HrCreatedResponse } from 'src/types';
+
+import { CreateHrDto } from './dto/create-hr.dto';
+import { HrService } from './hr.service';
+
+@Controller('hr')
+export class HrController {
+  constructor(private readonly hrService: HrService) {}
+
+  @HttpCode(HttpStatus.OK)
```

Comment:
Z HR-em będzie tak jak ze Studentem. Aktualizujemy jego dane, a nie tworzymy

Reply:
Wybacz, ale pomyliłem się :) Myśl zawsze czy to kto coś napiszę ma sens :)
HR-owca dodaje admin, w systemie nic jeszcze nie ma, więc będzie POST :)